client/src/app/shared/models/model-zoo-data-source/base-model-zoo-data-source.ts

```diff
@@ -150,11 +158,11 @@ export abstract class BaseModelZooDataSource<T, U = string> implements DataSourc
   }
 
   // reuse mat table data source implementation to handle edge cases
-  private _sortData(data: T[], sort: IModelZooSort): T[] {
+  private _sortData(data: T[], sort: IModelZooSort<T>): T[] {
     if (!sort) {
       return data;
     }
-    const active = sort.active;
+    const active = sort.field as string;
```

Comment:
We need to handle this case gracefully as according to `IModelZooSort` interface non `string` is a valid `field` type.